docs/book/src/user/quick-start.md

```diff
@@ -167,6 +198,114 @@ spec:
         cloud-provider: aws
 ```
 {{#/tab }}
+{{#tab Azure}}
+
+<aside class="note warning">
+
+<h1>Action Required</h1>
+
+These examples include environment variables that you should substitute before creating the resources.
```

Comment:
nit: provide an example of setting the variables and replacing them in the spec. 

e.g. 

```bash
export AZURE_SUBSCRIPTION_ID_B64="$(echo -n "$AZURE_SUBSCRIPTION_ID" | base64 | tr -d '\n')"
export AZURE_TENANT_ID_B64="$(echo -n "$AZURE_TENANT_ID" | base64 | tr -d '\n')"
export AZURE_CLIENT_ID_B64="$(echo -n "$AZURE_CLIENT_ID" | base64 | tr -d '\n')"
export AZURE_CLIENT_SECRET_B64="$(echo -n "$AZURE_CLIENT_SECRET" | base64 | tr -d '\n')"

expenv spec.yml | sponge spec.yml
```

Reply:
Added below. FWIW, I copied that warning message from the other providers so vSphere and OpenStack don't have  examples.